rest/admin_api.go

```diff
@@ -666,6 +666,7 @@ func (h *handler) putRole() error {
 
 func (h *handler) deleteUser() error {
 	h.assertAdminOnly()
+	h.setHeader("Content-Type", "application/json")
```

Comment:
I'm not sure the response body is JSON for this header to be set. There's no response body at all. Just a status code.